.clang-tidy

```diff
@@ -123,6 +130,7 @@ Checks: [
   '-readability-uppercase-literal-suffix',
   '-readability-use-anyofallof',
   '-readability-math-missing-parentheses',
+  '-readability-container-contains',
```

Comment:
Not critical but it would be super cool to use `.contains` where possible

Reply:
https://github.com/ClickHouse/ClickHouse/pull/79653